src/controller/NotificationController.js

```diff
@@ -0,0 +1,116 @@
+import dotenv from 'dotenv';
+import NotificationUtil from '../utilities/Notification';
+
+dotenv.config();
+
+
+/**
+ * @description class for controlling notifications
+ */
+class NotificationController {
+  /**
+   *@description opts into email notification
+
```

Comment:
I feel "subscribe for email notifications" is clearer and easier to understand. see line 12.